src/hooks/use-refresh.hook.ts

```diff
@@ -9,9 +9,9 @@ type UseHookRefresh = [
 export const useHookRefresh = (): UseHookRefresh => {
   const [internalFlag, setInternalFlag] = useState(false)
 
-  const handleRefresh = useCallback(() => {
+  const onHookRefresh = useCallback(() => {
```

Comment:
onRefresh?

Reply:
Handled in [5ddf33e](https://github.com/ajktown/wordnote/pull/158/commits/5ddf33e05136c5e01ce02ca05840754462e6215c)